### PR TITLE
Updated documentation for the new Go db/admin binary

### PR DESCRIPTION
**Pull request by rawlinp**
#### What does this PR do?

Updated documentation for the new Go db/admin binary

#### Which TC components are affected by this PR?

- [x] Documentation
- [ ] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

#### What is the best way to verify this PR?
Build the docs and check for errors/warnings.

#### Check all that apply

- [ ] This PR includes tests
- [x] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [ ] This PR includes a database migration (ensure that migration sequence is correct)
- [ ] This PR fixes a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2954/
Test PASSed.

File: docs/source/development/traffic_ops.rst [hunk 1]
```diff
@@ -229,30 +229,11 @@ To install the Traffic Ops Developer environment:
...
-		OK    20141222103718_extension.sql
-		OK    20150108100000_add_job_deliveryservice.sql
-		OK    20150205100000_cg_location.sql
-		OK    20150209100000_cran_to_asn.sql
-		OK    20150210100000_ds_keyinfo.sql
-		Seeding database...
-		Warning: Using a password on the command line interface can be insecure.
+#. Use the ``reset`` and ``upgrade`` commands of the ``admin`` binary (see `Database Management`_ for usage) to set up the traffic_ops database(s).
+
+	 Example Usage::
```
Comment (ocket8888) [inline docs/source/development/traffic_ops.rst]: The reason I left this as `Example Usage::` was because it included output and I didn't know all of the parsers available to me at the time. Your's is a shell command, so instead of just the generic `::` it'd be best if you could use `.. code-block:: shell` like so:
```rst
.. code-block:: shell
    :caption: Example Usage

    db/admin --env=development reset && db/admin --env=development upgrade
```

**Comment by ocket8888** [comment]
Ah, I see that now. That's good with me provided the new script fully implements all of those and only those.

File: docs/source/development/traffic_ops.rst (hunk 1 above)
Comment (ocket8888) [reply docs/source/development/traffic_ops.rst]: Not sure, I'll check it out.

File: docs/source/development/traffic_ops.rst [hunk 2]
```diff
@@ -229,30 +229,12 @@ To install the Traffic Ops Developer environment:
...
-		OK    20141222103718_extension.sql
-		OK    20150108100000_add_job_deliveryservice.sql
-		OK    20150205100000_cg_location.sql
-		OK    20150209100000_cran_to_asn.sql
-		OK    20150210100000_ds_keyinfo.sql
-		Seeding database...
-		Warning: Using a password on the command line interface can be insecure.
+#. Use the ``reset`` and ``upgrade`` commands of the ``admin`` binary (see `Database Management`_ for usage) to set up the traffic_ops database(s).
+
+	.. code-block::shell
```
Comment (ocket8888) [inline docs/source/development/traffic_ops.rst]: This is why you're not seeing this in the output. It's also why that Perl snippet isn't being rendered. `..` in RST starts a comment - as you probably know. Without the space between the `::` and the syntax name, the lexer detects that as just a plain word, not a directive. It needs to match `^\s*.. [\w\-]+:: .*$` to be considered a directive.

tl;dr: you're missing a space after `::` and before `shell`.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2963/
Test FAILed.

**Comment by rawlinp** [comment]
@ocket8888 ok, I believe everything is addressed, and the other code-block syntax errors I found have been fixed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/2965/
Test FAILed.
